web/packages/teleport/src/Navigation/NavigationItem.tsx

```diff
@@ -154,6 +158,26 @@ export function NavigationItem(props: NavigationItemProps) {
     []
   );
 
+  // showRedDot returns red dot component if the feature recommendation state is 'NOTIFY'
+  function showRedDot(featureName: NavTitle): JSX.Element {
```

Comment:
`showRedDot` sounds like a flag
```suggestion
  function renderHighlightFeature(featureName: NavTitle): JSX.Element {
```
or something like that